### ISSUE TITLE
step sequencer not working with newest bitwig updates

### ISSUE THREAD

**Issue by jhags8** [dplduffy/LaunchpadScriptV3#2]
hello! would you mind helping me out with this if possible? I'm using a Launchpad mini
thanks,
Joe

**Comment by dplduffy** [comment]
What seems to be the issue?

**Comment by jhags8** [comment]
when I go into the sequencer mode, the green square doesn't follow the time and therefore is not usable. I cannot add notes or function properly in this mode...... I think it may have to do with the newer updates.

**Comment by jhags8** [comment]
honestly thank you tho, I was about to give up. I will check back soon if you don't reply for a while, but i'll try to stay on.

**Comment by dplduffy** [comment]
I'm sorry it's not working for you. Unfortunately I have sold my launchpad mini a while ago so it may be difficult for me to help. Did you check the YouTube video I posted showing the functionality? Also the KVR thread may have some useful information.  https://www.kvraudio.com/forum/viewtopic.php?f=268&t=439866

**Comment by jhags8** [comment]
I've watched it. but I will check the KVR thread. Maybe you know some others who would be able to write a script for this if it is a script error?

**Comment by dplduffy** [comment]
There are a bunch of good coders on the KVR thread that may be able to help, unfortunately I don't know any names personally. When you say the green square doesn't follow the time, you are focused on the clip you want to play and the clip is playing correct?

**Comment by jhags8** [comment]
yes I tried creating the clip and while it is playing in recording mode, the green square in the sequencer mode does not follow the beats it just stays right where it is. I have the recording function on and I've messed with the overwrite and such. nothing happens

**Comment by jhags8** [comment]
it says you are from PA on the KVR thread... i'm from NY, could i mail this thing to you to use so you could fix it?

**Comment by jhags8** [comment]
i'm going to bed. but i'll check the thread sometime tomorrow! thank you for your help
